### DISCUSSION TITLE
uBO Lite for Safari?

### DISCUSSION THREAD

**Discussion by arjpar** [uBlockOrigin/uBOL-home#33] [Q&A]
According to [this comment](https://github.com/uBlockOrigin/uBlock-issues/issues/1444#issuecomment-1452620459) in an issue discussing the possibility of porting uBO Lite to Safari, many of the missing APIs were added. How feasible is it to port uBO Lite to Safari?  If possible, the only problem would be finding a maintainer since @gorhill [doesn't have access to a macOS system](https://github.com/uBlockOrigin/uBlock-issues/issues/1444#issuecomment-1252252758).

**Comment by gorhill** [comment]
https://testflight.apple.com/join/JjTcThrV
